src/wasm.h

```diff
@@ -1610,6 +1614,17 @@ class Module {
     }
     importsMap.erase(name);
   }
+
+  void removeImportsWithSubstring(Name name) {
```

Comment:
kind of the same concern, I'd rather keep this out of wasm.h. How about adding it to a header in `ast/`, perhaps something like `module.h` or `module_utils.h` (i'm not great at naming consistency myself)